### PR TITLE
🌿: introduce gem publishing

### PR DESCRIPTION
**Pull request by armandobelardo**

### CODE REVIEW

File: lib/gemconfig.rb [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
-    AUTHORS = [""].freeze
-    EMAIL = ""
-    SUMMARY = ""
-    DESCRIPTION = ""
-    HOMEPAGE = "https://github.com/REPO/URL"
-    SOURCE_CODE_URI = "https://github.com/REPO/URL"
-    CHANGELOG_URI = "https://github.com/REPO/URL/blob/master/CHANGELOG.md"
+    VERSION = "0.0.0"
+    AUTHORS = ["AssemblyAI"].freeze
+    EMAIL = "engineering.sdk@assemblyai.com"
```
Comment (Swimburger) [inline lib/gemconfig.rb:7]: Make this `rubygems@assemblyai.com`

File: lib/gemconfig.rb [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
+    SUMMARY = "AssemblyAI Ruby SDK"
+    DESCRIPTION = "AssemblyAI Ruby SDK"
```
Comment (Swimburger) [inline lib/gemconfig.rb:9]: How long can the description be? Here's what I got for the GH repo description:
```
The AssemblyAI Ruby SDK provides an easy-to-use interface for interacting with the AssemblyAI API, which supports async and real-time transcription, audio intelligence models, as well as the latest LeMUR models.
```

File: assemblyai.gemspec [hunk 3]
```diff
@@ -10,10 +10,9 @@ Gem::Specification.new do |spec|
...
   spec.description = AssemblyAI::Gemconfig::DESCRIPTION
   spec.homepage = AssemblyAI::Gemconfig::HOMEPAGE
-  spec.required_ruby_version = ">= 2.6.0"
+  spec.required_ruby_version = ">= 2.7.0"
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = AssemblyAI::Gemconfig::SOURCE_CODE_URI
-  spec.metadata["changelog_uri"] = AssemblyAI::Gemconfig::CHANGELOG_URI
   spec.files = Dir.glob("lib/**/*")
   spec.bindir = "exe"
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
```
Comment (Swimburger) [inline assemblyai.gemspec:18]: Can we add `spec.license = 'MIT'`?

**Review by Swimburger — CHANGES_REQUESTED** [review]

File: assemblyai.gemspec (hunk 3 above)
Comment (armandobelardo) [reply assemblyai.gemspec:18]: Definitely!

File: lib/gemconfig.rb (hunk 2 above)
Comment (armandobelardo) [reply lib/gemconfig.rb:9]: This works for sure!

File: lib/gemconfig.rb (hunk 1 above)
Comment (armandobelardo) [reply lib/gemconfig.rb:7]: Updated!

**Review by Swimburger — APPROVED** [review]

**Pull request merged**